docs/source/interfaces/direct/dns_powder_tof/DNS Powder TOF.rst

```diff
@@ -0,0 +1,38 @@
+.. _dns_powder_tof-ref:
+
+.. toctree::
```

Comment:
The help option in the interface is not pointing to this doc. I think it needs to be moved.